extensions/amp-list/0.1/amp-list.js

```diff
@@ -197,10 +197,20 @@ export class AmpList extends AMP.BaseElement {
    * @private
    * @return {?Element}
    */
-  getloadMoreButton_() {
+ getloadMoreButton_() {
     if (!this.loadMoreButton_) {
       this.loadMoreButton_ = childElementByAttr(
           this.element, 'load-more-button');
+
+      if(!this.loadMoreButton_) {
+        this.loadMoreButton_ = this.win.document.createElement('div');
+        this.loadMoreButton_.setAttributeNode(this.win.document.createAttribute('load-more-button'));
+
+        let innerButton = this.win.document.createElement('div');
```

Comment:
We should not create "buttons" using divs. This does not have the proper semantics for screen readers or handle keyboard interactivity correctly. We should use a `<button>` and reset styles as needed.